schemas/definitions.json

```diff
@@ -0,0 +1,348 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/FloEdelmann/open-fixture-library/master/schemas/definitions.json",
+
```

Comment:
Please add `"$comment": "Should not be used directly as a JSON schema."` or similar.

Reply:
Done.